notifications/NotificationBase.cpp

```diff
@@ -117,6 +117,7 @@ bool CNotificationBase::SendMessageEx(
 		fText = CURLEncode::URLEncode(fText);
 	}
 
+	boost::mutex::scoped_lock SendMessageEx(SendMessageExMutex);
```

Comment:
I wanted to keep the 'thread queue' in order, if multiple calls are made to SendMessageEx, I'm not sure what happens (haven't checked the code of all notifiers, if they're all protected individually, then were fine indeed). It might not be necessary, but I thought better safe than sorry?